eoglearn/datasets/eegeyenet.py

```diff
@@ -14,23 +15,31 @@ def _get_params(subject, run):
     row = df.loc[(df.subject == subject.upper()) & (df.run == int(run))]
     assert len(row) == 1
     row = row.T.squeeze()
+    task = row["task"]
     return dict(
                 url=row["url"],
-                archive_name=f"{subject}_DOTS{run}_EEG.mat",
-                folder_name=f"EEGEYENET-Data/dots/{subject}",
+                archive_name=f"{subject}_{task}{run}_EEG.mat",
+                folder_name=f"EEGEYENET-Data/{task}/{subject}",
```

Comment:
Should it be `f"EEGEYENET-Data/{task.lower()}/{subject}"` to be consistent with the previous code, or the case doesn't matter here?

Reply:
Oh thanks for bringing that up. It does matter, but I'm wondering if it is worth the change to keep the case consistent across the API. i.e., since we use "DOTS" and "AS" in the code, maybe we be consistent and name the folder "DOTS / "AS"? 

The next time you download the data you need to be aware of this (so that you don't keep a copy in both "dots" and "DOTS" directories).  WDYT? worth the change or no?

Reply:
Yes, fine with me. I was just not sure if that path was use for reading an existing repository (in which case it would crash) or to write up the data (which would be fine). From what you say, it seems to be the later.